tests/src/NPMBeanTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2013-2016 Eugene Petrenko
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.jonnyzzz.teamcity.plugins.node.common.NPMBean;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+@Test
+public class NPMBeanTest {
+
+
+  @Test
+  public void testParseCommands() throws Exception {
+    List<String[]> variants = Arrays.asList(
```

Comment:
I've tried to use data provider and test method with arguments (String, String[]) but failed miserably with random errors in testng intestines.